rescale_mode_percentile.py

```diff
@@ -0,0 +1,256 @@
+# coding=utf-8
+
+#################################
+#
+# Imports from useful Python libraries
+#
+#################################
+
+import numpy
+import scipy.ndimage
+import statistics
+
+#################################
+#
+# Imports from CellProfiler
+#
+##################################
+
+import cellprofiler.image
+import cellprofiler.module
+import cellprofiler.setting
+
+__doc__ = """\
+RescaleModePercentile
+=============
+**RescaleModePercentile** is an image processing module that rescales flourescent images.
```

Comment:
fluorescent

Reply:
Oops - things that happen if you are not a native speaker. Thank you

Reply:
No problem! I look a the spelling of that word a lot more frequently than you probably do :)